website/client/src/components/tasks/spells.vue

```diff
@@ -76,138 +89,224 @@
 </template>
 
 <style lang="scss" scoped>
-  .drawer-wrapper {
-    width: 100vw;
-    position: fixed;
-    bottom: 0;
-    left: 0;
-    z-index: 19;
-
-    .drawer-container {
-      left: auto !important;
-      right: auto !important;
-      min-width: 60%;
-    }
-  }
+.drawer-wrapper {
+  width: 100vw;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 19;
 
-  .drawer-slider {
-    margin-top: 1em;
+  .drawer-container {
+    left: auto !important;
+    right: auto !important;
+    min-width: 38em;
```

Comment:
<img width="543" alt="Schermata 2020-11-28 alle 19 39 04" src="https://user-images.githubusercontent.com/589911/100523662-5f6f4700-31b2-11eb-90dc-f7495bce6d89.png">

This seems to be a bit bigger than the design on zeplin, 456px instead of 408px, also we prefer to use rems not ems as units

Reply:
It's causing the skills not be completely centered